detection-rules/impersonation_employee_urgent_request.yml

```diff
@@ -10,7 +10,7 @@ source: |
 
   and sender.display_name in~ $org_display_names
 
-  and any([body.plain.raw, body.html.inner_text], 
+  and any([body.current_thread.text], 
```

Comment:
@morriscode this doesn't need an `any` anymore

Reply:
Roger that, sed didn't catch that one :) thanks!